problemsets-mat223/modules/module2.tex

```diff
@@ -389,7 +389,7 @@
     -2t + 1 &=  3s + 9
 \end{align*}
 in two unknowns  $s$ and $t$.  This is an {\it overdetermined\/}
-system, and it may or may not have a solution.
+system\index{overdetermined system}\index{system of linear equations!overdetermined system}, and it may or may not have a solution.
```

Comment:
For concepts that play a big role in the text, we might put them in the index twice (categorized and categorized), but `over-determined system` never gets mentioned again, so it's not a bit concept.